### PR TITLE
INT-3123 upgrade charts and iq version

### PR DESCRIPTION
**Pull request by jflinchbaugh**
upgrade to latest helm charts and bump to latest iq version.

JIRA: https://issues.sonatype.org/browse/INT-3123
Build: https://jenkins.ci.sonatype.dev/job/integrations/job/cloud/job/operator-iq/job/feature-snapshots/job/INT-3123-upgrade-charts/

### CODE REVIEW

File: deploy/olm-catalog/nxiq-operator-certified/1.95.0-1/nxiq-operator-certified.v1.95.0-1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,368 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (jflinchbaugh) [inline deploy/olm-catalog/nxiq-operator-certified/1.95.0-1/nxiq-operator-certified.v1.95.0-1.clusterserviceversion.yaml:1]: this is a copy of the previous CSV with new versions applied.

File: deploy/olm-catalog/nxiq-operator-certified/nxiq-operator-certified.package.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: nxiq-operator-certified.v1.94.0-4
-  name: alpha
-defaultChannel: alpha
+- currentCSV: nxiq-operator-certified.v1.95.0-1
+  name: stable
+defaultChannel: stable
```
Comment (jflinchbaugh) [inline deploy/olm-catalog/nxiq-operator-certified/nxiq-operator-certified.package.yaml:4]: bump to stable channel for future IBM certification.

File: helm-charts/iqserver/templates/NOTES.txt [hunk 3]
```diff
@@ -4,21 +4,19 @@ If you are using minikube, you can find the url as:
...
+Type:                     ClusterIP
 IP:                       1.2.3.4
-Port:                     <unset>  8070/TCP
+Port:                     application  8070/TCP
 TargetPort:               {{ .Values.iq.applicationPort }}/TCP
-NodePort:                 <unset>  31785/TCP
 Endpoints:                172.17.0.5:{{ .Values.iq.applicationPort }}
 Session Affinity:         None
-External Traffic Policy:  Cluster
 Events:                   <none>
```
Comment (jflinchbaugh) [inline helm-charts/iqserver/templates/NOTES.txt:22]: note updates (cluster ip) from latest yaml.

File: deploy/operator.yaml [hunk 4]
```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: nxiq-operator-certified
           # Replace this with the built image name
-          image: registry.connect.redhat.com/sonatype/nxiq-operator-certified:1.94.0-4
+          image: quay.io/jflinchbaugh/nxiq-operator-certified:1.95.0-1
```
Comment (kellyrob99) [inline deploy/operator.yaml:19]: leftovers from testing?

File: deploy/operator.yaml [hunk 5]
```diff
@@ -30,4 +30,4 @@ spec:
             - name: OPERATOR_NAME
               value: "nxiq-operator-certified"
             - name: RELATED_IMAGE_IQ
-              value: registry.connect.redhat.com/sonatype/nexus-iq-server:1.94.0-ubi-1
+              value: registry.connect.redhat.com/sonatype/nexus-iq-server:1.95.0-ubi-2
```
Comment (kellyrob99) [inline deploy/operator.yaml:33]: doesn't match the version information in the container image above

File: helm-charts/iqserver/templates/service.yaml [hunk 6]
```diff
@@ -10,4 +10,4 @@ spec:
     port: {{ .Values.iq.applicationPort }}
   - name: admin
     port: {{ .Values.iq.adminPort }}
-  type: NodePort
+  type: ClusterIP
```
Comment (jflinchbaugh) [inline helm-charts/iqserver/templates/service.yaml:13]: cluster ip from latest helm charts.

File: helm-charts/iqserver/templates/deployment.yaml [hunk 7]
```diff
@@ -18,6 +18,9 @@ spec:
       imagePullSecrets:
         - name: {{ template "iqserver.fullname" . }}-imagepull
       {{- end }}
+      {{- if .Values.deployment.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
+      {{- end }}
```
Comment (jflinchbaugh) [inline helm-charts/iqserver/templates/deployment.yaml:23]: fix from latest charts: terminationGracePeriodSeconds is back.

File: deploy/operator.yaml (hunk 5 above)
Comment (jflinchbaugh) [reply deploy/operator.yaml:33]: this is the ubi version of the app image. it had been deployed twice for whatever reason and latest is at *-2. the other version to which you are likely comparing this one is the version of the operator container.

File: deploy/operator.yaml (hunk 4 above)
Comment (jflinchbaugh) [reply deploy/operator.yaml]: yep. i also caught this while i was reviewing and annotating the PR.

File: deploy/olm-catalog/nxiq-operator-certified/1.96.0-1/nxiq-operator-certified.v1.96.0-1.clusterserviceversion.yaml [hunk 8]
```diff
@@ -0,0 +1,369 @@
...
+  - licensing
+  - policy
+  - components
+  links:
+  - name: Nexus Lifecycle
+    url: https://www.sonatype.com/product-nexus-lifecycle
+  maintainers:
+  - email: support@sonaype.com
+    name: Sonatype
+  maturity: stable
```
Comment (jflinchbaugh) [inline deploy/olm-catalog/nxiq-operator-certified/1.96.0-1/nxiq-operator-certified.v1.96.0-1.clusterserviceversion.yaml:365]: stable channel for ibm cert.

**Pull request merged**